### DISCUSSION TITLE
Incorrect links in the "Additional Responses in OpenAPI" aricle

### DISCUSSION THREAD

**Discussion by smiling-watermelon** [fastapi/fastapi#11262] [Questions] [answered]
I am not quite sure, where to report this, but I've found that links in [this section](https://fastapi.tiangolo.com/advanced/additional-responses/#more-information-about-openapi-responses) of the mentioned article are not leading exactly to a place they should.

Currently, they lead to:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#responsesObject
And
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#responseObject

But those sections should be linked like this:
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#responses-object
And
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#response-object

This is a very minor error, but I think it's still worth writing about it.

**Comment by svlandeg** [comment, marked as answer]
Thanks for the note! This has been fixed in this PR: https://github.com/fastapi/fastapi/pull/11996 🎉
